src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -46,7 +47,7 @@ public FeedbackResponseCommentRow(FeedbackResponseCommentAttributes frc, String
         this.sessionTimeZone = sessionTimeZone;
         this.createdAt = TimeHelper.formatDateTimeForSessions(frc.createdAt, this.sessionTimeZone);
         this.commentText = frc.commentText.getValue();
-        this.commentGiverName = getCommentGiverNameFromEmail(giverDisplay);
+        this.commentGiverName = SanitizationHelper.desanitizeIfHtmlSanitized(getCommentGiverNameFromEmail(giverDisplay));
```

Comment:
This also need to be removed after data migration right? If so, please add comment?